### DISCUSSION TITLE
Issue with dropdowns not populating using Combobox and Picker starter examples

### DISCUSSION THREAD

**Discussion by AvidDabbler** [adobe/react-spectrum#3406] [Q&A] [answered]
Hey all,
I am just getting started with Adobe spectrum and I am having an issue where none of the items that I have setup are displaying in my dropdowns. I have flowed the workflow of setting up my provider and installing requirements.

Currently my example is showing the dropdown combobox. One setup with `defaultItems` and the other with `<Item>` children, but both of them are showing empty dropdowns. I did some searching to see if this was a common problem in react and didn't come up with anything.

Has anyone run into the issue where dropdowns don't populate?

Example - https://codesandbox.io/s/elated-kare-u9my5x?file=/src/App.js

<img width="393" alt="image" src="https://user-images.githubusercontent.com/8471756/184888648-e0d446e8-4241-469c-8bed-20aaa31fc1d6.png">

**Comment by reidbarber** [comment, marked as answer]
Here's a fixed version: https://codesandbox.io/s/fixed-picker-combobox-sandbox-u0mv0g?file=/src/index.js

There were two issues:
1. Picker doesn't have a `defaultItems` prop, so I changed that to `items`.
2. React Strict Mode isn't supported yet, so I removed that in `index.js`. We have an issue for that here: https://github.com/adobe/react-spectrum/issues/779

**Reply by AvidDabbler** [reply]
Awesome! Thanks!

I am working on a Typescript project and just put this together as an example. Am I out of luck on using Typescript? Should i just convert to jsx components?

**Reply by reidbarber** [reply]
Typescript will work fine! This is referring to [React strict mode](https://reactjs.org/docs/strict-mode.html), which is different from typescript's strict mode.

**Reply by AvidDabbler** [reply]
amazing! thanks so much! my problem ended up being the `next.config.js` I needed to set strict to false:

```
const nextConfig = {
  reactStrictMode: false,
 ...
};

module.exports = withTM({...nextConfig});
```

**Comment by whiteWYN** [comment]
您好，邮件已收到。我会尽快回复。

**Reply by snowystinger** [reply]
From google translate, this appears to be spam. Did you mean to write something here? otherwise I can delete the comment.
